features/sidekiq.feature

```diff
@@ -77,7 +76,6 @@ Scenario Outline: A handled RuntimeError can be notified
   | ruby version | sidekiq_version | created_at_present |
   | 1.9.3        | ~> 2            | false              |
   | 2.0          | ~> 2            | false              |
-  | 2.0          | ~> 3            | true               |
```

Comment:
Why have these been removed?

Reply:
Added back in, but added fixed Gems to ensure compatibility.